### ISSUE TITLE
Export html without GUI in Plugin sourceview 

### ISSUE THREAD

**Issue by megrxu** [zim-desktop-wiki/zim-desktop-wiki#932]
There is an export issue when using the plugin sourceview without GUI:

```
ERROR: Exception while loading plugin: sourceview
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib/python3.7/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib/python3.7/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib/python3.7/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib/python3.7/site-packages/zim/plugins/sourceview.py", line 32, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib/python3.7/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
```

`PageViewExtension` is actually no need to import.

**Comment by megrxu** [comment]
Fixed in #933.

**Comment by MeisterP** [comment]
The same issue exists with a few other plugins as well.

```
ERROR: Exception while loading plugin: insertsymbol
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/insertsymbol.py", line 11, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
ERROR: Exception while loading plugin: versioncontrol
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/versioncontrol/__init__.py", line 29, in <module>
    from zim.gui.mainwindow import MainWindowExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/mainwindow.py", line 34, in <module>
    from zim.gui.pageview import PageView
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
ERROR: Exception while loading plugin: attachmentbrowser
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/attachmentbrowser/__init__.py", line 47, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
ERROR: Exception while loading plugin: spell
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/spell.py", line 17, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
ERROR: Exception while loading plugin: pageindex
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/pageindex/__init__.py", line 21, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
ERROR: Exception while loading plugin: linesorter
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/linesorter.py", line 14, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
ERROR: Exception while loading plugin: tags
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 360, in load_plugins_from_preferences
    self.load_plugin(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 461, in load_plugin
    klass = self.get_plugin_class(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/__init__.py", line 415, in get_plugin_class
    mod = get_module(modname)
  File "/usr/lib64/python3.6/site-packages/zim/utils.py", line 35, in get_module
    mod = __import__(name)
  File "/usr/lib64/python3.6/site-packages/zim/plugins/tags.py", line 16, in <module>
    from zim.plugins.pageindex import PageTreeStore, PageTreeStoreBase, PageTreeView, \
  File "/usr/lib64/python3.6/site-packages/zim/plugins/pageindex/__init__.py", line 21, in <module>
    from zim.gui.pageview import PageViewExtension
  File "/usr/lib64/python3.6/site-packages/zim/gui/pageview.py", line 3471, in <module>
    CURSOR_TEXT = Gdk.Cursor.new_from_name(Gdk.Display.get_default(), 'text')
TypeError: Argument 0 does not allow None as a value
```
